crates/router/src/db/merchant_connector_account.rs

```diff
@@ -398,6 +455,15 @@ impl MerchantConnectorAccountInterface for MockDb {
         }
     }
 
+    async fn find_merchant_connector_account_by_merchant_id_connector_name(
+        &self,
+        _merchant_id: &str,
+        _connector_name: &str,
+        _key_store: &domain::MerchantKeyStore,
+    ) -> CustomResult<domain::MerchantConnectorAccount, errors::StorageError> {
+        Err(errors::StorageError::MockDbError.into())
```

Comment:
Please implement this, literally everything else in this interface has been implemented. This is needed for us to run integration tests.